src/js/components/Form/__tests__/__snapshots__/Form-test.js.snap

```diff
@@ -2073,7 +2071,7 @@ exports[`Form uncontrolled 2`] = `
               class="StyledTextInput-sc-1x30a0s-0 cMMVpP"
               name="test"
               placeholder="test input"
-              value="v"
+              value=""
```

Comment:
The value is now missing, used to be 'v'.

Reply:
This is because the test is using an uncontrolled input element, in which case we shouldn't be setting the value explicitly, just letting the element track it internally.